engine/src/main/java/org/terasology/rendering/nui/layers/ingame/ExtraMenuScreen.java

```diff
@@ -0,0 +1,45 @@
+package org.terasology.rendering.nui.layers.ingame;
```

Comment:
Add the standard license header please, grab it from just about any Java file and make sure the year is 2019 :-)